File: codeforces/158A/c/158a.c
#include <stdio.h>

#define MAX	64

int main()
{
	int n, k, i;
	int scores[MAX];

	scanf("%d %d", &n, &k);
	for (i = 0; i < n; i++) {
		scanf("%d", &scores[i]);
	}

	for (i = 0; i < n; i++) {
		if (scores[i] <= 0 || scores[i] < scores[k-1])
			break;
	}
	printf("%d\n", i);

	return 0;
}

File: codeforces/158A/go/158a.go
package main
import "fmt"

const MAX_LEN int = 64;

func main() {
	var n, k int
	var scores[MAX_LEN] int

	fmt.Scanf("%d %d\n", &n, &k)
	for i := 0; i < n; i++ {
		fmt.Scanf("%d", &scores[i])
	}

	var r int
	for r = 0; r < n; r++ {
		if scores[r] <= 0 || scores[r] < scores[k-1] {
			break
		}
	}

	fmt.Printf("%d\n", r)
}
